photon-core/src/main/java/org/photonvision/vision/target/TrackedTarget.java

```diff
@@ -66,6 +66,8 @@ public TrackedTarget(
         this.m_mainContour = origTarget.m_mainContour;
         this.m_subContours = origTarget.m_subContours;
         this.m_shape = shape;
+        Point[] corners = shape.contour.getConvexHull().toArray();
+        m_targetCorners = List.of(corners);
```

Comment:
I'm like pretty sure the order that this happens in is OK. Can you confirm that doing this won't overwrite corners detected by like cornerdetectionpipe? Glancing thru the code this looks fine

Reply:
Unfortunately I didn't manage to simulate the changes on my rpi/computer